CSharp/BatchExplorer/Models/ModelBase.cs

```diff
@@ -106,13 +106,17 @@ private List<PropertyModel> ObjectToPropertyModelRecursive(object o, List<string
                         //Don't track null properties in order to keep the list short
                         //propertyModel = new SimplePropertyModel(propInfo.Name, string.Empty);
                     }
+                    else if ((propertyType == typeof(TimeSpan) || propertyType == typeof(TimeSpan?)) && propertyValue.Equals(TimeSpan.MaxValue))
+                    {
+                        propertyModel = new SimplePropertyModel(propInfo.Name, "Unlimited");
```

Comment:
@itowlson Can you rebase/squash away your other 4 commits ("merge remote tracking branch...")?  5 commits for a 2 line change seems excessive.